### ISSUE TITLE
[Bug Report] Identify Crashing Stash

### ISSUE THREAD

**Issue by nicholasfoote** [stashapp/stash#2060]
**Describe the bug**
When running the Identify task (on default settings), after several minutes, it crashes the application. Crashes at the exact same point every time, but I've tried deleting the last file it processed, and it fails on the next one.
In the docket logs, I see this repeated a few times:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0xf3f0e4]
```

**To Reproduce**
Go to Tasks
Click Identify
Wait 5-10 minutes

**Expected behavior**
Identify has been working fine up until about a week ago

**Screenshots**
![image](https://user-images.githubusercontent.com/10204046/143299540-4aca8917-4487-413e-a5f7-32fdbb17ca7e.png)

**Stash Version: (from Settings -> About):**
v0.11.0-29-gff32f22c

**Desktop (please complete the following information):**
 - OS: UNRAID Docker
 - Browser: Firefox
 - Version: 94.02

**Comment by nicholasfoote** [comment]
I guess it was a specific file it was being caught on each time. Deleted it and good to go.

**Comment by WithoutPants** [comment]
Reopening because this is certainly a problem within stash.

**Comment by SmallCoccinelle** [comment]
There should be a fix for this in #2043 at the end.
